player/lua/auto_profiles.lua

```diff
@@ -88,7 +88,11 @@ function get(name, default)
     if not watched_properties[name] then
         watched_properties[name] = true
         mp.observe_property(name, "native", on_property_change)
-        cached_properties[name] = mp.get_property_native(name)
+        local err = nil
+        cached_properties[name], err = mp.get_property_native(name)
```

Comment:
Move the call to `get_property_native` to before the call to `mp.observe_property`, and break out early of the function.

Reply:
Also undo the addition of the true value to watched_properties.

Reply:
imo it's better when `watched_properties` gets set, otherwise it'll call `mp.get_property_native()` every time that function gets called for that property.

Reply:
I changed it so it's not added to `cached_properties` nor is `observe_property` attempted. This does actually prevent redundant `evalute` calls so it's worth it. I left it in `watched_properties` to avoid duplicate `mp.get_property_native` calls however.